bin/run.js

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env node
 
+if (process.argv.includes('--debug')) {
+  process.env.DEBUG = 'sf*';
+  process.env.SF_LOG_COLORIZE = 'false';
+  process.env.SF_LOG_STDERR = 'true';
+}
```

Comment:
this needs `SF_LOG_LEVEL=trace` for logs to be rendered:
https://github.com/salesforcecli/cli/blob/00be8bca7e22eed7adcf8d4fc5a7ac2bf6bd60ab/src/flags.ts#L36